.env

```diff
@@ -4,7 +4,7 @@ NEXT_PUBLIC_API_URL=https://beta.api.opensauced.pizza/v2
 NEXT_PUBLIC_EXP_API_URL=https://beta.api.opensauced.pizza/v2
 NEXT_PUBLIC_POSTHOG_ID=phc_Y0xz6nK55MEwWjobJsI2P8rsiomZJ6eZLoXehmMy9tt
 NEXT_PUBLIC_CLOUD_NAME=dgxgziswe
-NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=test
+NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_test_51H1yp3JnZ7CYWRG2KmQMIE8sizoaSaZ8FxlEsc5uQb81gVswDlWu6FErmcukqc6Hso3mhCWgnLsjWphight4t1Mf00jZZjckyZ
```

Comment:
Revert this change. You can create a `.env.local` file at the root of the project next to the `.env` and put your local environment variable changes there. Its not picked up by source control, but Next.JS will pick them up when running the app